tqdm/tests/tests_tqdm.py

```diff
@@ -116,8 +116,11 @@ def write(self, s):
         self.text += s
         self.cursor = len(self.text)
```

Comment:
Could you please update this line BTW to this:

`self.cursor += len(s)`

Because else it could be buggy if we seek and then write.